app/views/teacher_training_adviser/steps/_overseas_timezone.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= f.govuk_fieldset legend: { text: 'You told us you live overseas' } do %>
+
+<p class="govuk-body">You need to speak to us and should book a call in advance. We will contact you once your call is booked.</p>
+<p class="govuk-body">You must have the details of your overseas qualifications when we contact you.</p>
+
+<%= f.govuk_phone_field :telephone, width: 20,
+  label: { text: 'Contact telephone number *' } %>
+
+<div class="govuk-form-group">
+  <%= f.label 'Select your time zone', class: "govuk-label" %>
+  <%= f.time_zone_select :time_zone, ActiveSupport::TimeZone.all.drop(1), {}, { class: "govuk-select" }  %>
```

Comment:
Why do we `drop(1)` from the list of timezones?

Reply:
001 > ActiveSupport::TimeZone.all.first
 => #<ActiveSupport::TimeZone:0x00007fc2dd627cb0 @name="International Date Line West", @utc_offset=nil, @tzinfo=#<TZInfo::DataTimezone: Etc/GMT+12>>

Reply:
Ah, I see, in that case is it worth putting a method in the model along the lines of:

```
def time_zones_sans_utc
  ActiveSupport::TimeZone.all.drop(1)
end
```

```
f.object.time_zones_sans_utc
```

Then we can write a test for it? feels like we're relying on `ActiveSupport` ordering them a certain way which may change in a future update.